src/less/widgets/mwstew.ui.PageHeadSectionWidget.less

```diff
@@ -17,8 +17,8 @@
 
 	&-title {
 		position: relative;
-		top: ~"calc(" @pageHeadSectionHeight / 2 ~" - 4em)";
-		text-shadow: rgba(0, 0, 0, 0.6) 0 1px 2px;
+		top: ~'calc(' @pageHeadSectionHeight / 2 ~' - 4em)';
```

Comment:
I think I was trying to make it responsive, while getting the title to appear properly above the section. I can definitely do it with a wrapper whose width is responsive, though. I'll add that to my todo list.